Core/clim-core/panes.lisp

```diff
@@ -1808,9 +1807,9 @@ which changed during the current execution of CHANGING-SPACE-REQUIREMENTS.
   (dolist (child (alexandria:ensure-list contents))
     (sheet-adopt-child sheet child)))
 
-(defmethod compose-space ((bboard bboard-pane) &key width height)
+(defmethod compose-space ((bboard bboard-pane) &key (width 100) (height 100))
   (declare (ignore width height))
```

Comment:
width and height are not ignored anymore, please remove the declaration.

Reply:
(Done, sorry about that.)